test/functional/ex_cmds/wviminfo_spec.lua

```diff
@@ -0,0 +1,62 @@
+local helpers, lfs = require('test.functional.helpers'), require('lfs')
+local clear, execute, eq, neq, spawn, nvim_prog, set_session, wait =
+  helpers.clear, helpers.execute, helpers.eq, helpers.neq, helpers.spawn,
+  helpers.nvim_prog, helpers.set_session, helpers.wait
+
+-- Lua does not have a sleep function so we use the system command.  If the
+-- command does not support sub second precision we use math.floor() to get
+-- full seconds.
+local sleep = function(millisec)
+  local sec = millisec / 1000
+  local round = math.floor(sec)
+  if round == 0 then round = 1 end
+  os.execute('sleep '..sec..' || sleep '..round)
+end
+
+describe(':wviminfo', function()
+  local file = 'foo'
+  before_each(function()
+    clear()
+    os.remove(file)
+  end)
+
+  it('creates a file', function()
+    -- TODO
+    -- Set up the nvim session to be able to write viminfo files.  Is it
+    -- possible to do this outside of the it() call?
+    local nvim2 = spawn({nvim_prog, '-u', 'NONE', '--embed'})
+    --local nvim2 = spawn({nvim_prog, '-u', 'NONE', '--embed', '--cmd', 'let hans=42' })
+    set_session(nvim2)
+    --eq(43, eval('hans'))
+
+    -- Assert that the file does not exist.
+    eq(nil, lfs.attributes(file))
+    execute('wv! '..file)
+    wait()
+    -- Assert that the file does exist.
+    neq(nil, lfs.attributes(file))
```

Comment:
> but the "explanation" is never shown in the test output

Are you sure? That would be a major problem. If I change `~=` to `==` at this line: https://github.com/neovim/neovim/commit/1f0830f7004c60f899fc169baecd68960f26ea70#diff-d587b4ef5d4b7947c66765d46ff9a341R50

The test results are:

```
Failure → test/functional/ex_cmds/wviminfo_spec.lua @ 32
:wviminfo overwrites existing files
test/functional/ex_cmds/wviminfo_spec.lua:50: wviminfo_test should be a viminfo-formatted file 
```

>  I don't think it is a good idea to use an argument which I was unable to find in the busted docs.

It's actually a stock lua function. Personally I find it annoying that busted doesn't provide an "explanation" parameter for all of their assert functions. I assume it has to do with BDD dogma about `it()` blocks being small and self-explanatory :P  But if a comment is needed to explain an assertion, then evidently that spec is not self-explanatory (and in practice this is quite common, and not reasonable to avoid it).